save-backend/src/main/kotlin/com/saveourtool/save/backend/service/ProjectService.kt

```diff
@@ -70,7 +72,14 @@ class ProjectService(
      * @param name
      * @param organizationName
      */
-    fun findByNameAndOrganizationName(name: String, organizationName: String) = projectRepository.findByNameAndOrganizationName(name, organizationName)
+    fun findByNameAndOrganizationName(name: String, organizationName: String) = projectRepository.findByNameAndOrganizationName(name, organizationName).getOrNull()
```

Comment:
maybe change in Repository to `Project?`?

Reply:
I need it to return `Optional` because of reactivity in `LnkContestProjectController`

Reply:
Done